app/contributor-preferences.html

```diff
@@ -258,6 +258,30 @@
       <div>
         <hr>
       </div>
+      <div>
+        <span data-i18next="contributorPreferences.opacityTheme"></span>
+      </div>
+      <div>
+        <p data-i18next="contributorPreferences.opacityInfo"></p>
+      </div>
+      <div>
+        <label data-i18next="contributorPreferences.opacity" data-same-width-label for="opacity"></label>
+        <input type="range" min="10" max="100" step="10" list="tickmarksOpacity" name="opacity" data-divisor="0.01" id="opacity">
```

Comment:
Could we make a step 5? 10 seems to small... And also make min 5 as well? Pretty much I think I would make it the same as Volume so that it's consistent.